br/pkg/task/stream.go

```diff
@@ -932,7 +932,17 @@ func RunStreamRestore(
 	g glue.Glue,
 	cmdName string,
 	cfg *RestoreConfig,
-) error {
+) (err error) {
+	startTime := time.Now()
+	defer func() {
+		dur := time.Since(startTime)
+		if err != nil {
+			summary.Log(cmdName+" failed summary", zap.Error(err))
```

Comment:
when using restore point with full backup storage. will this print two summary log?

Reply:
The summary log here will not print twice repeatedly, but will print both full restore log and point restore log.